langgraph/prebuilt/__init__.py

```diff
@@ -1,9 +1,51 @@
 """langgraph.prebuilt exposes a higher-level API for creating and executing agents and tools."""
-from langgraph.prebuilt import chat_agent_executor
-from langgraph.prebuilt.agent_executor import create_agent_executor
-from langgraph.prebuilt.chat_agent_executor import create_react_agent
-from langgraph.prebuilt.tool_executor import ToolExecutor, ToolInvocation
-from langgraph.prebuilt.tool_node import ToolNode, tools_condition
+
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from langgraph.prebuilt import chat_agent_executor
+    from langgraph.prebuilt.agent_executor import create_agent_executor
+    from langgraph.prebuilt.chat_agent_executor import create_react_agent
+    from langgraph.prebuilt.tool_executor import ToolExecutor, ToolInvocation
+    from langgraph.prebuilt.tool_node import ToolNode, tools_condition
+    from langgraph.prebuilt.tool_validator import ValidationNode
+
+
+def __getattr__(name: str) -> Any:
```

Comment:
Why do we need this?

Reply:
ha i'll remove apologies - meant to have that as a separate pr